Formula/grokj2k.rb

```diff
@@ -36,39 +34,27 @@ class Grokj2k < Formula
   uses_from_macos "perl"
   uses_from_macos "zlib"
 
-  on_macos do
-    # HACK: this should not be a test dependency but is due to a limitation with fails_with
-    depends_on "llvm" => [:build, :test] if DevelopmentTools.clang_build_version >= 1205
```

Comment:
We'll probably need LLVM on older macOS. I think upstream is using C++20. They mentioned that a minimum of Clang 12 which would be Xcode 13.

Reply:
Given that it requires GCC 10 on Linux, this is very likely because of needing C++20.